core/src/main/java/org/infinispan/remoting/transport/jgroups/JGroupsTransport.java

```diff
@@ -1641,4 +1705,42 @@ private enum SiteUnreachableReason {
       SITE_DOWN_EVENT,
       SITE_UNREACHABLE_EVENT
    }
+
+   private InfinispanSpan createSpan(ReplicableCommand command) {
+      // self conscious defensive programming
+      if (!tracingEnabled) {
+         return null;
```

Comment:
Returning `DisabledInfinispanSpan.instance()` here and in some places below would remove all the `null` checks on the requests.

Reply:
you're right... even I'm going to remove this redundant block

Reply:
removed!